java/src/jmri/jmrix/dccpp/DCCppPredefinedMeters.java

```diff
@@ -68,16 +122,21 @@ public void message(DCCppMessage m) {
         // Do nothing
     }
     
+    /* dispose of all defined meters             */
+    /* NOTE: I don't know if this is ever called */
     public void dispose() {
```

Comment:
For your info. The comment is fine.

`dispose()` is probably not called now, but I'm not entirely sure. But it might be called later. Possible scenario:
1. Table for Meters is implemented in JMRI which allows meters to be removed
2. DCCpp gets a feature to enable/disable the meters in DCCpp.
3. The user has DCCpp meters activated, and therefore the meters get created.
4. The user disables meters in DCCpp.
5. The user removes the DCCpp predefined meters in the JMRI Meters table.

Reply:
Hm. I removed all meters and it still wasn't called.

Reply:
Ooops. It was some time ago I did this so I don't remember. But I think dispose() should be called if the meter is removed from the manager. It could be a bug.